proto/vectorindex.proto

```diff
@@ -6,6 +6,7 @@ service VectorIndex {
     rpc UpsertItemBatch(_UpsertItemBatchRequest) returns (_UpsertItemBatchResponse) {}
     rpc DeleteItemBatch(_DeleteItemBatchRequest) returns (_DeleteItemBatchResponse) {}
     rpc Search(_SearchRequest) returns (_SearchResponse) {}
+    rpc SearchAndFetchVector(_SearchAndFetchVectorsRequest) returns (_SearchAndFetchVectorsResponse) {}
```

Comment:
Should the rpc be called `SearchAndFetchVector**s**`?

Reply:
Sharp eyes! Yes! https://github.com/momentohq/client-protos/pull/229/commits/d0d4271d68423d42fbb467981e8884b1523b66d2